model/src/main/kotlin/config/CopyrightGarbage.kt

```diff
@@ -32,6 +33,11 @@ data class CopyrightGarbage(
      */
     @JsonSerialize(converter = StringSortedSetConverter::class)
     val items: Set<String> = emptySet()
-)
+) {
+    /**
+     * Return whether the statment of the given [finding] is a garbage.
+     */
+    operator fun contains(finding: CopyrightFinding): Boolean = finding.statement in items
```

Comment:
There is no need to make this depend on the `CopyrightFinding` class.
Could you please change the parameter to `String` type, for ease of use from random contexts.